tools/install/installFromBuildkite.sh

```diff
@@ -3,6 +3,9 @@
 # Exit on any error
 set -e
 
+echo "Sorry, this script needs to be updated to download APKs from GitHub action. Buildkite is not building APKs anymore."
```

Comment:
Will the script be updated in another dedicated PR?

Reply:
I guess I will write another script to install from GitHub, but yes, in another PR. I will write it during the next release.

Reply:
(Actually I am working on it right now)